### DISCUSSION TITLE
Impossible from the start

### DISCUSSION THREAD

**Discussion by adderallisjustspeed** [3kh0/ext-remover#257] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
In both of the methods one needs to access  https://chrome.google.com/webstorex, but this website is blocked for me too ):
Any help?

**Comment by xHydra1337** [comment]
I think it depends on what version you are running, most likely wont work but you can also try google.com/chromexx

**Comment by ILikeBypassing** [comment, marked as answer]
try adding some more letters to the url for example https://chrome.google.com/webstorenevergonnagiveyouup
or use my crash exploit here https://github.com/3kh0/ext-remover/discussions/232

also LTBEEF doesnt work on chromeos version 106 and above so check your chrome://version
